Integration test failure in CI: nimbus/test_remote_settings.py::test_create_new_experiment_approve_remote_settings

Seeing failures similar to this across most PRs right now:
```
FAILED nimbus/test_remote_settings.py::test_create_new_experiment_approve_remote_settings[KLAR]
```
Example: https://app.circleci.com/pipelines/github/mozilla/experimenter/11318/workflows/a406755f-6671-47dd-a6f6-39e965dcb099/jobs/34275

--- a/docker-compose-prod.yml
+++ b/docker-compose-prod.yml
@@ -52,12 +52,12 @@ services:
       - db_volume:/var/lib/postgresql
 
   redis:
-    image: redis
+    image: redis:6.2.6
     ports:
       - "6379:6379"
 
   kinto:
-    image: mozilla/kinto-dist
+    image: mozilla/kinto-dist:23.3.1
     environment:
       KINTO_INI: /etc/kinto.ini
     ports:
@@ -68,7 +68,7 @@ services:
       - ./kinto/server.ini:/etc/kinto.ini
 
   autograph:
-    image: mozilla/autograph
+    image: mozilla/autograph:4.1.1
     ports:
       - "8000:8000"
 

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -105,12 +105,12 @@ services:
       - db_volume:/var/lib/postgresql
 
   redis:
-    image: redis
+    image: redis:6.2.6
     ports:
       - "6379:6379"
 
   kinto:
-    image: mozilla/kinto-dist
+    image: mozilla/kinto-dist:23.3.1
     environment:
       KINTO_INI: /etc/kinto.ini
     ports:
@@ -121,7 +121,7 @@ services:
       - ./kinto/server.ini:/etc/kinto.ini
 
   autograph:
-    image: mozilla/autograph
+    image: mozilla/autograph:4.1.1
     ports:
       - "8000:8000"
 

